src/mon/MonClient.cc

```diff
@@ -111,14 +107,17 @@ int MonClient::get_monmap_privately()
 
   ldout(cct, 10) << "have " << monmap.epoch << " fsid " << monmap.fsid << dendl;
 
+  std::random_device rd;
```

Comment:
I wasn't suggesting using a non-library function, but this isn't worth holding up this pr for.